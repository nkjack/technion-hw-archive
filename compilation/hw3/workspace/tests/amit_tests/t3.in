// no main
bool preFunc(int x, int y)
@pre (x >= 1)
@pre(y >= 1)
{
    return x < y;
}

int return6()
@pre(preFunc(1, 2))
{
    return 6;
}
