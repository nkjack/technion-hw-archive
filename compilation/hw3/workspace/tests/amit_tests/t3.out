---end scope---
preFunc - 2 preconditions
x INT -1
y INT -2
---end scope---
return6 - 1 preconditions
Program has no 'void main()' function
